### DISCUSSION TITLE
【サポーター募集】OSS Gateオンラインワークショップ2023-04-29

### DISCUSSION THREAD

**Discussion by daipom** [oss-gate/workshop#1610] [Announcements]
4月29日は、OSS Gateワークショップ開催日です！

サポーター参加者を募集中ですが、みなさまのご都合はいかがでしょうか？
サポーターとビギナーのペア参加もお待ちしています！
もし、OSS開発を盛り上げていきたい開発者の方や、OSS開発に参加してみたいけど一人じゃ不安な方がお知り合いにいたらぜひ紹介してください！

詳しい情報は、下記イベントページとIssueをご覧ください。

* https://oss-gate.doorkeeper.jp/events/145991
* #1589

「サポーター」の役割は、「ビギナー」の不安をやわらげて、最初の一歩を一緒にすすんでみることです。過去に「サポーター」だった方はもちろんのこと、「ビギナー」として参加したことがあれば、**十分その役割を果たすことができます！**

「自分も過去にビギナーで参加して不安だったんだけど、実際にやってみたらそんなに怖がることはなかったですよ。一緒にやっていましょう！」と声をかけて「サポーター」が背中を押してくれれば、「ビギナー」の人の不安はやわらぎますよね。

サポーターや進行役が当日どのようなことをするかについては、以下のページに詳しい紹介があります。
初めての方でも、分からない所はサポートメンターがお手伝いしますので、ぜひ勇気を出してチャレンジして頂ければ幸いです！

サポーターガイド https://oss-gate.github.io/workshop/supporter-guide.html
進行役スタートガイド https://oss-gate.github.io/workshop/new-to-moderator.html

進行役を務めてみたい方は、サポーターとして参加登録をされた上で、「進行役をしてみたい」とコメントして頂けましたら幸いです。

たくさんの人のご参加お待ちしています！

**Comment by daipom** [comment]
進行役やりまーす

**Comment by daipom** [comment]
直前にビギナーさんが一人参加希望されていて、キャンセル待ちになっているのですが、明日のワークショップにサポーター参加できそうな方はいらっしゃいませんかね

**Reply by otegami** [reply]
他に参加できそうな方がいらっしゃらなかったら、サポーターで参加したいです🙋‍♀️
- おそらく大丈夫だとは思うのですが、新しい環境からアクセスするためネットワークが少し心配なためmm
- 今日の夜 9 時には現地で確認できるのでネットワークに問題がありそうでしたら相談させてください

**Reply by daipom** [reply]
ありがとうございます！！

分かりました！もし環境などの問題がなさそうならよろしくお願いします！

**Reply by otegami** [reply]
遅くなりました。
今確認してみたところ問題なさそうでしたので、明日問題なく参加できそうです🙏

**Reply by daipom** [reply]
ありがとうございます！
あしたよろしくおねがいします！

**Comment by daipom** [comment]
DONE!
